R/btw_client.R

```diff
@@ -87,11 +87,17 @@
 #'   `client` argument, then the `btw.client` R option, and finally the `btw.md`
 #'   project file, using only the client definition from the first of these that
 #'   is available.
-#' @param tools Optional names of tools or tool groups to include in the chat
-#'   client. By default, all btw tools are included. For example, use
-#'   `include = "docs"` to include only the documentation related tools, or
-#'   `include = c("env", "docs")`, etc. `btw_client()` also supports
-#'   `tools = FALSE` to skip registering \pkg{btw} tools with the chat client.
+#' @param tools A list of tools to include in the chat, defaults to
+#'   [btw_tools()]. Join [btw_tools()] with additional tools defined by
+#'   [ellmer::tool()] to include additional tools in the chat client.
+#'   Alternatively, you can use a character values to refer to specific btw
+#'   tools by name or by group. For example, use `tools = "docs"` to include
+#'   only the documentation related tools, or `tools = c("env", "docs")` to
+#'   include the environment and documentation tools, and so on. You can also
+#'   refer to btw tools by name, e.g. `tools = "btw_tool_docs_help_page"` or
+#'   alternatively in the shorter form `tools = "docs_help_page"`. Finally,
+#'   set `tools = FALSE` to skip registering \pkg{btw} tools with the chat
+#'   client.
```

Comment:
My reflex here would be to just tell folks to pass `list()` in this case to reduce the possible Ways This Argument Can Be Passed and also so that we don't have to carve out for that in `flatten_and_check_tools()`

Reply:
Just to make sure I'm seeing that you're calling out the lines I think you are: I think you're referring to `tools = FALSE`. We have that case mostly for the YAML spec, mostly because I like

```yaml
tools: false
```

better than

```yaml
tools: []
```

Reply:
Ohhhh I understand. That makes sense!